src/queryCache.js

```diff
@@ -474,6 +475,15 @@ export function makeQueryCache() {
       query.scheduleStaleTimeout()
     }
 
+    query.clear = () => {
+      clearTimeout(query.staleTimeout)
+      clearTimeout(query.cacheTimeout)
+      if (query.cancelQueries) {
```

Comment:
We could use the new `query.cancel()` method here :)

Reply:
added. I notcied that `query.cancel` isn't in the readme api or in the `.d.ts` definition, which is probably why I missed it...